catalog/admin/module/catalog-admin-module-maplayers/src/main/webapp/map-layers/index.js

```diff
@@ -142,13 +155,30 @@ const ProviderEditor = ({ provider, onUpdate, buffer, onEdit, error = Map() }) =
     <div style={{ padding: '0 16px' }}>
       <Flexbox flex='1'>
         <Checkbox
-          checked={typeof provider.get('proxyEnabled') === 'boolean' ? provider.get('proxyEnabled') : ''}
           label='Proxy Imagery Provider URL'
+          checked={bool(provider.get('proxyEnabled'))}
           onCheck={(e, value) => {
             onUpdate(value, 'proxyEnabled')
           }} />
       </Flexbox>
     </div>
+    <div style={{ padding: '0 16px' }}>
+      <Flexbox flex='1'>
+        <Flexbox width='250px'>
+          <Checkbox
+            label='Use Stored Credentials'
```

Comment:
`Allow Cross Origin Credentials` might be more clear here

Reply:
The more I think about, the more I think that this doesn't have anything to do with CORS at all (although that's the error I was getting in my browser during testing), especially after reading @millerw8 's description https://github.com/codice/ddf/pull/2496#pullrequestreview-70053008. So `Allow Cross Origin Credentials` also doesn't sound right.

Reply:
What about `Allow Credential Forwarding` with the warning message `Requests will fail if the server does not prompt for credentials`?

Reply:
👍

Reply:
Done